tracer/src/Datadog.Trace/Span.cs

```diff
@@ -174,6 +174,7 @@ public override string ToString()
             sb.AppendLine($"Duration: {Duration}");
             sb.AppendLine($"End: {StartTime.Add(Duration).ToString("O")}");
             sb.AppendLine($"Error: {Error}");
+            sb.AppendLine($"TraceSamplingPriority: {Context.TraceContext.SamplingPriority}");
```

Comment:
I think this is good, but there is the chance that it isn't set at this point
Maybe it'd be good to check if it has a value and if it doesn't log something like ("not set")?

Reply:
It'll log `null` if it's not set at this point, right? Can certainly null coalesce here 😄